superset-frontend/src/components/DvtDragCard/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Icon from '../Icons/Icon';
+import {
+  StyledDvtCard,
+  StyledDvtCardCard,
+  StyledDvtCardIcon,
+  StyledDvtCardLabel,
+} from './dvt-drag-card.module';
+
+export interface DvtDragCardProps {
+  label: string;
+  value: string;
+  icon: string;
+}
+
+const DvtDargCard = ({ label, value, icon }: DvtDragCardProps) => {
+  const handleDragStart = (e: React.DragEvent<HTMLDivElement>) => {
+    e.dataTransfer.setData(
+      'text/plain',
+      JSON.stringify({ label, value, icon }),
```

Comment:
text/plain aslında "drag-drop" diyelim  drag drop için uğraşıyoruz demek bir ismi verelim daha iyi,

{ label, value, icon } aslında tek (value) yeterli value bir object item için